composer.json

```diff
@@ -10,23 +10,25 @@
         }
     ],
     "require": {
-        "php": "^8.0 || ^8.1",
+        "php": "^8.0",
         "php-task/php-task": "^2.0",
-        "symfony/http-kernel": "^5.4 || ^6.0",
-        "symfony/dependency-injection": "^5.4 || ^6.0",
-        "symfony/expression-language": "^5.4 || ^6.0",
-        "symfony/config": "^5.4 || ^6.0",
-        "symfony/console": "^5.4 || ^6.0",
-        "symfony/process": "^5.4 || ^6.0",
+        "symfony/http-kernel": "^5.4 || ^6.0 || ^7.0",
+        "symfony/dependency-injection": "^5.4 || ^6.0 || ^7.0",
+        "symfony/expression-language": "^5.4 || ^6.0 || ^7.0",
+        "symfony/config": "^5.4 || ^6.0 || ^7.0",
+        "symfony/console": "^5.4 || ^6.0 || ^7.0",
+        "symfony/process": "^5.4 || ^6.0 || ^7.0",
         "doctrine/orm": "^2.5.3"
     },
     "require-dev": {
-        "symfony/framework-bundle": "^5.4 || ^6.0",
-        "symfony/finder": "^5.4 || ^6.0",
-        "symfony/yaml": "^5.4 || ^6.0",
-        "symfony/phpunit-bridge": "^5.4 || ^6.0",
+        "symfony/framework-bundle": "^5.4 || ^6.0 || ^7.0",
+        "symfony/finder": "^5.4 || ^6.0 || ^7.0",
+        "symfony/yaml": "^5.4 || ^6.0 || ^7.0",
         "doctrine/doctrine-bundle": "^1.5 || ^2.0",
-        "doctrine/data-fixtures": "^1.3.3"
+        "doctrine/data-fixtures": "^1.3.3",
+        "phpunit/phpunit": "^9.6.5 || ^10",
```

Comment:
PHPUnit 11 is not supported yet by Symfony because of a error / exception handler registration conflict